monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -103,6 +103,10 @@ def is_interrupted(self):
             self.exploit_result.error_message = "Exploiter has been interrupted"
         return self.interrupt.is_set()
 
+    class InterruptError(Exception):
+        # Raise when exploiter gets interrupted
+        pass
+
```

Comment:
This class should be under `common/utils/exceptions.py`, if we decide to go this route.

Reply:
It's not common. Nothing outside of exploiters uses it. Things should go in `common/` if, and only if, they're used (or expected to be used) by both agent and island